collect_weather_data_from_ghcn_bigquery.py

```diff
@@ -0,0 +1,89 @@
+"""
+Collect Weather Data from GHCN 
+Author: Francisco Vannini 
+Date: 2020-01-29
+
+Requires: 
+Google Project - For more information see : https://cloud.google.com/bigquery/docs/authentication/
+
+Inputs:
```

Comment:
I appreciate these comments and I think they would be more helpful if you encapsulate this into functions with their own docstrings. It would help with organization, maintainability, and readability of your code.

A simple organization method would just to have a `main()` function that is called. You can see more here: https://www.guru99.com/learn-python-main-function-with-examples-understand-main.html

Reply:
I will look into it. Thanks for the feedback!